### PR TITLE
feat: add silent mode

### PR DESCRIPTION
**Pull request by JonZeolla**
# Contributor Comments

This adds a new `SILENT` mode, which turns off all of the `_feedback` calls, meant to provide context to users as they are running the tool.  This is useful if they want to use tools like `jq` or `sed` on the output of the command that they're running.

## Pull Request Checklist

Thank you for submitting a contribution to our project!

In order to streamline the review of your contribution we ask that you review and comply with the below requirements:

- [x] Rebase your branch against the latest commit of the target branch.
- [x] If you are adding a dependency, please explain how it was chosen.
- [x] If manual testing is needed in order to validate the changes, provide a testing plan and the expected results.
- [x] If there is an issue associated with your Pull Request, link the issue to the PR.
- [x] Validate that documentation is accurate and aligned to any project updates or additions.

Don't forget our more detailed contribution guidelines
[here](https://github.com/SeisoLLC/operations/blob/main/documents/software-guidelines.md#contributing-to-a-repository).

### CODE REVIEW

File: docs/Logging/index.rst [hunk 1]
```diff
@@ -53,7 +53,7 @@ If you'd like to run ``terraform validate`` on terraform stored in your current
 ``AWS_ACCESS_KEY_ID``, ``AWS_SECRET_ACCESS_KEY``, and any other AWS environment variables (including ``AWS_SESSION_TOKEN`` if you are assuming a role)
 environment variables as defined `here<https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html>_` and run the following::
 
-    docker run --env-file <(env | grep -E '^CW_|^AWS_') -v $(pwd)/fluent-bit.cw_example.conf:/usr/local/etc/fluent-bit/fluent-bit.outputs.conf seiso/easy_infra:latest terraform validate
+    docker run --env-file <(env | grep -E '^CW_|^AWS_') -v $(pwd)/fluent-bit.cw_example.conf:/usr/local/etc/fluent-bit/fluent-bit.outputs.conf seiso/easy_infra:latest-terraform terraform validate
```
Comment (JonZeolla) [inline docs/Logging/index.rst:56]: Bugfix

File: docs/Logging/index.rst [hunk 2]
```diff
@@ -66,4 +66,12 @@ The contents of ``fluent-bit.cw_example.conf`` here are as follows::
         Auto_create_group  true
 
 For more details on the fluent-bit Amazon CloudWatch output plugin, including features like cross account role assumption, see `this page
-<https://docs.fluentbit.io/manual/pipeline/outputs/cloudwatch>`_.
\ No newline at end of file
+<https://docs.fluentbit.io/manual/pipeline/outputs/cloudwatch>`_.
+
+easy_infra output
```
Comment (JonZeolla) [inline docs/Logging/index.rst:71]: Docs for this new feature

File: docs/Logging/index.rst [hunk 3]
```diff
@@ -66,4 +66,12 @@ The contents of ``fluent-bit.cw_example.conf`` here are as follows::
...
+-----------------
+
+``easy_infra`` sends some contextual information to stdout/stderr. If you want to disable this output, pass in the environment variable ``SILENT`` with a value
+of ``true``, for instance:
+
+    docker run -e SILENT=true seiso/easy_infra:latest-terraform /bin/bash -c "terraform output | jq .example.value"
```
Comment (JonZeolla) [inline docs/Logging/index.rst:77]: I tested this in my local

File: Taskfile.yml [hunk 4]
```diff
@@ -19,6 +19,11 @@ includes:
       PYTHON_VERSION: '{{.PYTHON_VERSION}}'
       VERSION: '{{.VERSION}}'
 
+  terraform:
```
Comment (JonZeolla) [inline Taskfile.yml:22]: Minor tweak to make use of the new centralized goat logic

**Review by ChiefHolland — APPROVED** [review]
LGTM

**Pull request merged**